diff --git a/lib/google/ads/google_ads/v1/resources/detail_placement_view_pb.rb b/lib/google/ads/google_ads/v1/resources/detail_placement_view_pb.rb
index 2eac7d0..05de7b4 100644
--- a/lib/google/ads/google_ads/v1/resources/detail_placement_view_pb.rb
+++ b/lib/google/ads/google_ads/v1/resources/detail_placement_view_pb.rb
@@ -17,6 +17,6 @@ Google::Protobuf::DescriptorPool.generated_pool.build do
   end
 end
 
-module Google::Ads::GoogleAds::V1::Services
+module Google::Ads::GoogleAds::V1::Resources
   DetailPlacementView = Google::Protobuf::DescriptorPool.generated_pool.lookup("google.ads.googleads.v1.resources.DetailPlacementView").msgclass
 end
