diff --git a/lib/google/ads/google_ads/v1/resources/group_placement_view_pb.rb b/lib/google/ads/google_ads/v1/resources/group_placement_view_pb.rb
index 95fd352..7d80431 100644
--- a/lib/google/ads/google_ads/v1/resources/group_placement_view_pb.rb
+++ b/lib/google/ads/google_ads/v1/resources/group_placement_view_pb.rb
@@ -16,6 +16,6 @@ Google::Protobuf::DescriptorPool.generated_pool.build do
   end
 end
 
-module Google::Ads::GoogleAds::V1::Services
+module Google::Ads::GoogleAds::V1::Resources
   GroupPlacementView = Google::Protobuf::DescriptorPool.generated_pool.lookup("google.ads.googleads.v1.resources.GroupPlacementView").msgclass
 end
